### PR TITLE
Zero downtime migration tests

### PR DESCRIPTION
**Pull request by triffer**
**Description**

Changes proposed in this pull request:

- Add script that execute migration integration tests and verifies zero downtime

**Pre-Merge Checklist**

- [ ] As a PR reviewer, verify code coverage and evaluate if it is acceptable.

**Related issues**

- https://github.com/kyma-project/api-gateway/issues/1301

### CODE REVIEW

File: tests/integration/testsuites/ory/scenario_migration_jwt_v1beta1.go [hunk 1]
```diff
@@ -70,7 +71,7 @@ func (s *scenario) resourceOwnedByApiRuleDoesNotExist(resourceKind string) error
 			return err
 		}
 
-		if len(list.Items) < 1 {
+		if len(list.Items) > 0 {
 			return fmt.Errorf("expected at least one %s owned by APIRule, got %d", resourceKind, len(list.Items))
```
Comment (mluk-sap) [inline tests/integration/testsuites/ory/scenario_migration_jwt_v1beta1.go:75]: The message is wrong, we expect that it **does not** exist

**Review by mluk-sap — DISMISSED** [review]

File: docs/contributor/04-30-ci-cd.md [hunk 2]
```diff
@@ -69,6 +69,7 @@ The following CI jobs are part of the release cycle.
...
 | [`PR Integration Release / Custom domain GCP`](https://github.com/kyma-project/api-gateway/blob/3d10f7c7801f514524a267239924fdc889f2be3f/.github/workflows/pull-integration-release.yaml#L88)                     | Executes the integration test suites that verify the functional correctness of Kyma API Gateway Operator on a Gardener GCP cluster. |
 | [`PR Integration Release / Custom domain AWS`](https://github.com/kyma-project/api-gateway/blob/3d10f7c7801f514524a267239924fdc889f2be3f/.github/workflows/pull-integration-release.yaml#L107)                    | Executes the integration test suites that verify the functional correctness of the API Gateway Operator on a Gardener AWS cluster.  |
+| [`PR Integration Release / Zero Downtime Migration Tests`]()                    | Executes the tests that verify the zero downtime migration from APIRule v1beta1 to v2alpha1 on a k3d cluster.  |
```
Comment (nataliasitko) [inline docs/contributor/04-30-ci-cd.md:72]:
```suggestion
| [`PR Integration Release / Zero Downtime Migration Tests`]()                    | Executes the tests that verify the zero downtime migration from APIRule `v1beta1` to `v2alpha1` on a k3d cluster.  |
```

**Review by nataliasitko — APPROVED** [review]
